generate_postmortem.sh

```diff
@@ -1457,83 +1462,83 @@ for NAMESPACE in $NAMESPACE_LIST; do
             fi
         done
     fi
-    #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-done
```

Comment:
@subyssurendran666 Could you please explain why is this loop removed here and added in Line 1539 ?

Reply:
@subindani The requirement is to collect all the normal data for any namespace in addition to cluster service version, install plans and subscriptions. That's why I have moved the codes to find the csv, ip and subscription information inside the for loop.

Reply:
Thank you @subyssurendran666